src/ussd_nurse.js

```diff
@@ -377,6 +376,89 @@ go.app = function() {
             });
         });
 
+        self.add('state_save_nursereg', function(name) {
+        
+            var registrant_update_info = {
+                "details": {
```

Comment:
@imsickofmaps with identity patch, should we pass through the whole updated identity (excl. auto fields like "url", "created_at", "updated_at", "created_by", "updated_by", "user") or only the fields that have changed..?